internal/rbd/rbd_attach.go

```diff
@@ -397,6 +399,30 @@ func waitForrbdImage(ctx context.Context, backoff wait.Backoff, volOptions *rbdV
 	return err
 }
 
+// statergicActionOnLogFile act on log file based on cephLogStrategy.
+func statergicActionOnLogFile(ctx context.Context, logStrategy, logFile string) {
+	switch logStrategyLookup[logStrategy] {
+	case compress:
+		go func(logFile string) {
+			var err error
+			if err = log.GzipLogFile(logFile); err != nil {
+				log.WarningLog(ctx, "failed to compress logfile: %s, error: %v", logFile, err)
+			}
+			if err = os.Remove(logFile); err != nil {
+				log.WarningLog(ctx, "failed to remove logfile: %s, error: %v", logFile, err)
```

Comment:
why compress also delete @pkalever ?

Reply:
The GzipLogFile() file creates a new file which will be a compressed format of the source and it expects the user to delete the original one!

Reply:
@humblec @nixpanic @Madhu-1  should we remove the original file inside  GzipLogFile() then ?

Reply:
yeah! Moved `os.Remove` part into GzipLogFile(), so that this looks clean and avoid confusion. Thanks!